### ISSUE TITLE
Metrics Carousel not longer working

### ISSUE THREAD

**Issue by stephanepapin** [SmartHalo-User-Group/SmartHalo-2#41]
## Description
Metrics Carousel not longer working

## Steps To Reproduce
try tap Metrics Carousel : no responses

## Configuration
* Phone type: Samsung Note 20 5G
* Phone OS version: Android 11
* SmartHalo version: 1.17.1

## Expected behavior
Changing function

## Actual behavior
Metrics Carousel frozen

## Additional context
WALKAROUND :
I've plug it 5 seconds (or wait until you got loading message) to power, unplug : all features back. Metrics Carousel working fine.

**Comment by smarthalo** [comment]
Yes, that's probably a crash. The workaround for now is to plug in the SH2 into a USB to reboot.

**Comment by stephanepapin** [comment]
> Yes, that's probably a crash. The workaround for now is to plug in the SH2 into a USB to reboot. 
Reset or reboot don't exist, response for support (Jessica) :

> Non, vous n'avez aucun moyen de le réinitialiser.

**Comment by nzwaneveld** [comment]
Connecting your SH2 to a charger is the reset. It is not a workaround.
